### DISCUSSION TITLE
Deploy without Operator

### DISCUSSION THREAD

**Discussion by chilly-dude** [strimzi/strimzi-kafka-operator#4639] [Q&A]
Hi all, Im relatively new to Kafka and using Strimzi, and Kubernetes Operators. Im working on a project for a client to deliver a containerised Kafka cluster and Ive been playing with Strimzi which seems like the ideal solution.

The target environment is Openshift, and unfortunately installing any Operators is not an option at the moment - I don't think we can get a cluster admin to do this right now. As far as Im aware the Operator is just a deployment pattern but also helps with state of pods, so is it possible to install Strimzi without using the Operator by installing Helm charts? Would we lose out on anything by doing this - e.g. statefulness of the brokers, topic. Any advice or help would be gratefully received. TIA

**Comment by scholzj** [comment]
I'm afraid Strimzi is an operator. So you cannot install or use it without operator. You do not necessarily need cluster-admin privileges. But you indeed need to have some cluster level rights such as installing CRDs etc. 

If you target OpenShift 4, you can also try whether you can install applications from OperatorHub which again is not necessarily bound to being cluster admin - but it depends on the cluster admin who they actually allowed to use it.

**Reply by chilly-dude** [reply]
Yeah thats my issue sadly, I dont think I'll be able to get any sort of cluster privileges. Fair enough, I will use this as ammunition to get the operator installed. I've already proven how simple it is to use so hopefully they can find time to look into it. Thanks for the response.
